app/views/group_expenses/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<p style="color: green"><%= notice %></p>
+
+<h1>Group expenses</h1>
+
+<div id="group_expenses">
+  <% @group_expenses.each do |group_expense| %>
+    <%= render group_expense %>
+    <p>
+      <%= link_to "Show this group expense", group_group_expense_path(id:group_expense.id) %>
+    </p>
+  <% end %>
+</div>
```

Comment:
- [ ] You did a good job working on this project! 💯 However, there are still some things that could be improved:

Please consider following the design guidelines for this view:

|Template|Your design|
|---|---|
| ![image](https://github.com/Elhambasir/budget-app/assets/77069011/63cb60e7-123e-4b00-b662-207b542d7809) | ![image](https://github.com/Elhambasir/budget-app/assets/77069011/6f28fe35-62b4-4005-96d5-8ffc11753736) |

Please consider using the full width of the device so the design could work fine on different types of mobile devices. Also, the button should be green and fixed to the bottom! Please consider applying this 🤓 🔧